src/components/SignIn.js

```diff
@@ -125,21 +125,36 @@ export const SignIn = hh(class SignIn extends Component {
     return user;
   };
 
+  renderSpinnerIfDisabled = (disabled) => {
+    return disabled ?
+      div({ style: { 'position': 'absolute', 'top': '45px', 'right': '45px', 'zIndex': '10000' } }, [
+        img({ src: '/images/loading-indicator.svg', alt: 'spinner' })
+      ]) :
+      h(GoogleLogin, {
+        scope: 'openid email profile',
+        theme: 'dark',
+        clientId: this.state.clientId,
+        onSuccess: this.responseGoogle,
+        onFailure: this.forbidden,
+        disabledStyle: { 'opacity': '25%', 'cursor': 'not-allowed' },
+      });
+  };
+
   render() {
 
     let googleLoginButton;
 
     if (this.state.clientId === '') {
-      googleLoginButton = div({ style: { 'position': 'relative', 'marginTop': '20px', 'marginLeft': '45px', 'zIndex': '10000' } }, [
+      googleLoginButton = div({ style: { 'position': 'absolute', 'top': '45px', 'right': '45px', 'zIndex': '10000' } }, [
```

Comment:
Suggestion. This aligns the spinner under the `Already Registered?` text and when the google button displays, that text jumps to the left. If we used this style: 
```
{ textAlign: 'center', height: '44px', width: '180px' }
```
Then the text stays in the same place and the spinner is replaced with the button in the exact same position.

Reply:
Yeah, I was going for more visual "clarity" by having the elements jump a bit as they reposition into place when the button loads, but I'm happy to realign the spinner!

Reply:
Thank you for the changes!